### ISSUE TITLE
TextGraphics.getWidth=0 in TextBox.repaint()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Valyrox/lanterna#131]
```
Hi,

I get exceptions due to TextGraphics being 0 in TextBox.repaint() method.

As a solution, I had to change

  if(displayString.length() > graphics.getWidth())

to

  if(displayString.length() > graphics.getWidth() && graphics.getWidth()>0);


Lanterna 2.1.9

Regards,
Aydin
```

Original issue reported on code.google.com by `ako...@gmail.com` on 17 Apr 2015 at 8:22

**Comment by GoogleCodeExporter** [comment]
```
That should be fixed now on release/2.1 branch
```

Original comment by `mab...@gmail.com` on 28 Jun 2015 at 12:09
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue has been closed because Google Code is shutting down shortly and all 
issues have been migrated, along with the rest of the project, to GitHub. 
However, notifications will probably not work when updates are made to this 
issue on GitHub so for future updates, please visit 
https://github.com/mabe02/lanterna/issues and subscribe to further updates 
there.

Thank you,
Martin
```

Original comment by `mab...@gmail.com` on 12 Jul 2015 at 7:39
- Changed state: **Duplicate**
